sdk/loganalytics/microsoft-azure-loganalytics/pom.xml

```diff
@@ -122,8 +122,6 @@
                 <artifactId>maven-compiler-plugin</artifactId>
                 <version>3.10.1</version> <!-- {x-version-update;org.apache.maven.plugins:maven-compiler-plugin;external_dependency} -->
                 <configuration>
-                    <source>1.7</source>
-                    <target>1.7</target>
```

Comment:
Any reason for removing the source and target versions? Wouldn't this change the baseline to Java 8?

Reply:
Yes, it would change the baseline. We can keep this if you think we should.

Reply:
Yes, these are track 1 libraries and we should keep the baseline the same (i.e. Java 7 in this case)